engine/runtime-compiler/src/main/scala/org/enso/compiler/pass/analyse/alias/graph/Graph.scala

```diff
@@ -318,12 +318,17 @@ object Graph {
     *                       Note that there may not be a link for all these definitions.
     */
   sealed class Scope(
-    var childScopes: List[Scope]                  = List(),
-    var occurrences: Map[Id, GraphOccurrence]     = HashMap(),
-    var allDefinitions: List[GraphOccurrence.Def] = List()
+    private[Graph] var _childScopes: List[Scope]                  = List(),
```

Comment:
I don't understand the purpose of this refactoring. Why not simply define these fields as `private var childScopes: List[Scope] = ...` without the rename and define public plain old getters?

Reply:
I'm only guessing but it looks like he wanted to limit the scope of variables so that one does not accidentally mutate them? But yes, the motivation is unclear.

Reply:
The title of this PR is:

>  Don't expose setters for Scope vars.

Renaming the fields to `var _xyz` and adding `def xyz = _xyz` achieves that goal. If there is a better way in Scala to have public getters, but private setters, feel free to propose it.